colcon_core/shell/template/prefix.bat.em

```diff
@@ -53,6 +57,36 @@ goto:eof
 goto:eof
 
 
+:: Get the package names in topological order
+:: using semicolons as separators and avoiding leading separators.
+:: first argument: the name of the result variable
+:: second argument: the base path to look for packages
+:ament_get_ordered_packages
```

Comment:
is it a mistake that this is `ament`?

Reply:
Certainly a mistake - good catch! Fixed in 930136de6aed28004b6ea79f8982a99f3c014dc3.